emailApp/mailAPI/MailSendingAPI.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from "react";
+import { KEYS } from "../keys/keys";
+import Tts from "react-native-tts";
+const fetch = require("node-fetch");
+
+export default class MailSendingAPI extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: "",
+    };
+  }
+  async emailControls1(url, headers, body) {
+    await fetch(url, {
+      method: "POST",
+      headers: headers,
+      body: JSON.stringify(body),
+    })
+      .then((res) => {
+        return res.json();
+      })
+      .then((result) => console.log(result));
+  }
+  async getDraftID(url, headers) {
+    await fetch(url, {
+      method: "GET",
+      headers: headers,
+    })
+      .then((res) => {
+        return res.json();
+      })
+      .then((result) => this.setState({ id: result.drafts[0].id }));
+  }
+
+  async componentDidMount() {
+    const { base64Message } = this.props;
+    console.log(base64Message);
+    const token = `Bearer ${KEYS.access_TOKEN}`;
+
+    const url =
+      "https://www.googleapis.com/gmail/v1/users/" + KEYS.EMAIL + "/drafts";
```

Comment:
Move "url" to constants file.

Reply:
You can even use Object.freeze (It makes an object immutable, i.e. you cannot change its properties.)